pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/steps/identity-step/identity-step.component.html

```diff
@@ -610,11 +610,11 @@ <h3 class="modal-title">Verify LDAP Configuration</h3>
                                             <clr-icon aria-live="assertive" a11yTooltipTrigger shape="info-circle" size="22">
                                             </clr-icon>
                                             <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
-                                                <span>A group name used to test LDAP group search functionality</span>
+                                                <span>Enter the OU attribute for the group to test LDAP group search functionality.</span>
                                             </clr-tooltip-content>
                                         </clr-tooltip>
                                     </label>
-                                    <input clrInput formControlName="testGroupName" placeholder="people" name="testGroupName" />
+                                    <input clrInput formControlName="testGroupName" placeholder="OU" name="testGroupName" />
```

Comment:
Could we make the placeholder OU attribute something like `testGroup` or `Group's OU Attribute`?